places-solution-integrations/target/attach-data.md

```diff
@@ -0,0 +1,76 @@
+
+Next, click the "Add" button under the "Events" section:
+
+<img src="../../.gitbook/assets/integrations/target/ad-addEvent.png" width="800" /><br />
+
+Select `Adobe Target` from the "Extension" dropdown, then select `Content Requested` from the "Event Type" dropdown. No additional configuration is necessary, so hit the "Keep Changes" button.
+
+<img src="../../.gitbook/assets/integrations/target/ad-setEvent.png" width="800" /><br />
+
+## Add Conditions
+
+If you wish to add `Conditions` to your rule, complete this step. If you don't, skip to [Define the Action](#define-the-action).
+
+In this example, we will add a condition that will cause our rule to trigger only for users that have launched the app five or more times.
+
+Click the "Add" button under the "Conditions" section:
+
+<img src="../../.gitbook/assets/integrations/target/ad-addCondition.png" width="800" /><br />
+
+Select `Mobile Core` from the "Extension" dropdown, and choose `Launches` from the "Condition Type" dropdown. In the UI on the right, modify the dropdown and number controls so the condition reads "User has launched the app greater than or equal to 5 times". When you're done, hit the "Keep Changes" button.
+
+<img src="../../.gitbook/assets/integrations/target/ad-setCondition.png" width="800" /><br />
+
+## Define the Action
+
+The last thing we need to do is define the `Action`. First, click the "Add" button under the "Actions" section:
+
+<img src="../../.gitbook/assets/integrations/target/ad-addAction.png" width="800" /><br />
+
+Select `Mobile Core` from the "Extension" dropdown and choose `Attach Data` from the "Action Type" dropdown.
+
+In the UI on the right, you have the ability to add a freeform JSON payload that will add data to an SDK event before any extension listening for this event will hear it. In our example, we are adding `mboxparameters` to all requests in this event before the Adobe Target extension process it. This results in each of the requests from this event having the new `mboxparameters` added on the outgoing Target request.
+
+In the example below, we are adding "poiCity" and "poiName" values to the `mboxparameters` for each request processed in the Target event. The values for the new keys will be determined dynamically by the SDK at the time this event processes:
+
+<img src="../../.gitbook/assets/integrations/target/ad-setAction.png" width="800" /><br />
+
+{% hint style="info" %}
+This JSON payload uses a special notation for the `"request"` object. In the original event, `"request"` is an array of anonymous objects. When attaching data to all of objects in an array using Attach Data, using the `[*]` notation on a key that is known to contain an array will cause the payload to be applied to all objects within that array.
+
+The notation of `"request[*]"` can be read out loud as "for each object in the `request` array".
```

Comment:
Please replace the quotation marks in the "read out loud" text with asterisks. This way it displays in italics like a message.